src/amo/components/SearchForm/Suggestion.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+/* eslint react/no-array-index-key: 0 */
+import React from 'react';
+import escapeStringRegexp from 'escape-string-regexp';
+
+type Props = {|
+  search: string,
+  suggestion: string,
+|};
+
+const Suggestion = ({ search, suggestion }: Props) => {
+  let parts;
+  if (search.replace(/\s+/, '') === '') {
+    parts = [suggestion];
+  } else {
+    // split the suggestion by the search typed by the user so that it can be
```

Comment:
Ahhhh, I see what this is doing now. I think using a word like "highlight" over identify will help illustrate what this is doing. A tweak to the docs could be:

```js
// Separate the existing search query text from the suggested search term
// returned from the autocomplete API. This allows us to highlight the search
// query already typed by the user in the text using the `.match-search` className
// below.
```

I'll admit I still don't understand what "since template strings are not by default" means.